providers/fips/fipsprov.c

```diff
@@ -753,23 +774,24 @@ int fips_intern_provider_init(const OSSL_CORE_HANDLE *handle,
                               const OSSL_DISPATCH **out,
                               void **provctx)
 {
-    OSSL_FUNC_core_get_library_context_fn *c_get_libctx = NULL;
+    OSSL_FUNC_core_get_library_context_fn *c_internal_get_libctx = NULL;
 
     for (; in->function_id != 0; in++) {
         switch (in->function_id) {
         case OSSL_FUNC_CORE_GET_LIBRARY_CONTEXT:
-            c_get_libctx = OSSL_FUNC_core_get_library_context(in);
+            c_internal_get_libctx = OSSL_FUNC_core_get_library_context(in);
             break;
         default:
             break;
         }
     }
 
-    if (c_get_libctx == NULL)
+    if (c_internal_get_libctx == NULL)
         return 0;
 
     if ((*provctx = PROV_CTX_new()) == NULL)
         return 0;
+
     /*
      * Using the parent library context only works because we are a built-in
      * internal provider. This is not something that most providers would be
```

Comment:
Aren't we supposed to be calling `c_internal_get_libctx` below?

Reply:
Err maybe - this has probably been like this for quite some time..

Reply:
No - it was correct before your changes. Your changes renamed `c_get_libctx` to `c_internal_get_libctx` everywhere except in the section below.

Reply:
No - it was correct before your changes. Your changes renamed c_get_libctx to c_internal_get_libctx everywhere except in the section below.